js/index.js

```diff
@@ -8,23 +8,58 @@ var path = require('path');
 var pkgInfo = require('../package.json');
 
 var DEFAULT_IGNORE = ['node_modules'];
+var SOURCEMAPS_OPTIONS = {
+  ignore: '--ignore',
+  ignoreFile: '--ignore-file',
+  noSourceMapReference: '--no-sourcemap-reference',
+  stripPrefix: '--strip-prefix',
+  stripCommonPrefix: '--strip-common-prefix',
+  validate: '--validate',
+  urlPrefix: '--url-prefix',
+  ext: '--ext'
+};
 
 var binaryPath =
   os.platform() === 'win32'
     ? path.resolve(__dirname, '..\\bin\\sentry-cli.exe')
     : path.resolve(__dirname, '../sentry-cli');
 
-function transformIgnore(ignore) {
-  if (Array.isArray(ignore)) {
-    return ignore
+function transformOption(option, values) {
+  if (Array.isArray(values)) {
+    return values
       .map(function(value) {
-        return ['--ignore', value];
+        return [option, value];
       })
       .reduce(function(acc, value) {
         return acc.concat(value);
       }, []);
   }
-  return ['--ignore', ignore];
+  return [option, values];
+}
+
+function normalizeOptions(options) {
+  var newOptions = [];
+  return Object.keys(SOURCEMAPS_OPTIONS).reduce(function(acc, sourceMapOption) {
```

Comment:
You don't need `newOptions` here, as this is what `acc` is for in `reduce` functions :)

Reply:
ESLint doesn't like that, that is how I did it before, had to change it because ESLint as you can see in the commit history

Reply:
```js
if (typeof options[sourceMapOption] !== 'undefined') {
  if (
    sourceMapOption === 'ignore' ||
    sourceMapOption === 'stripPrefix' ||
    sourceMapOption === 'stripCommonPrefix'
  ) {
    return newOptions.concat(
      transformOption(SOURCEMAPS_OPTIONS[sourceMapOption], options[sourceMapOption])
    );
  } else if (sourceMapOption === 'validate') {
    return newOptions.concat([SOURCEMAPS_OPTIONS[sourceMapOption]]);
  } else {
    return newOptions.concat(
      SOURCEMAPS_OPTIONS[sourceMapOption],
      options[sourceMapOption]
    );
  }
}
return newOptions;
```